### DISCUSSION TITLE
有在offline场景下，通过milvus-operator部署milvus集群成功的吗？

### DISCUSSION THREAD

**Discussion by mhen2531** [milvus-io/milvus#30053] [Ideas & Feature requests]
1、公司的环境没有连外网
2、cpu架构x86
3、cert-manage和milvus-operator,通过线下拉去镜像部署成功了
4、安装官网步骤，deploy milvus集群的时候，拉去下来的yaml文件里面的spec等都是空的，直接部署报错了

报错信息：error: SchemaError(io.milvus.v1alpha1.MilvusCluster.spec.components.indexNode.initContainers): array should have exactly one sub-item

**Comment by LoveEachDay** [comment]
@mhen2531 确保安装milvus 的manifest 文件是完整的

**Reply by mhen2531** [reply]
[按照官网-Deploy a Milvus cluster
：](https://milvus.io/docs/install_cluster-milvusoperator.md)
`kubectl apply -f https://raw.githubusercontent.com/zilliztech/milvus-operator/main/config/samples/milvus_cluster_default.yaml
`
拉取的yaml。里面都是空的
`apiVersion: milvus.io/v1beta1
kind: Milvus
metadata:
  name: my-release
  labels:
    app: milvus
spec:
  mode: cluster
  dependencies: {}
  components: {}
  config: {}
`
这块改怎么改呢？
